hosts/ghaf-log/configuration.nix

```diff
@@ -46,4 +58,47 @@
       efiInstallAsRemovable = true;
     };
   };
+
+  # Grafana
+  services.grafana = {
+    enable = true;
+
+    settings = {
+      server = {
+        http_port = 3000;
+        http_addr = "127.0.0.1";
+      };
+
+      # disable telemetry
+      analytics = {
+        reporting_enabled = false;
+        feedback_links_enabled = false;
+      };
+
+      # allow read-only access to dashboards without login
+      # this is fine because the page is only accessible with vpn
+      "auth.anonymous".enabled = true;
+    };
+
+    provision.datasources.settings.datasources = [
+      {
+        name = "loki";
+        type = "loki";
+        isDefault = true;
+        url = "http://${config.services.grafana.settings.server.http_addr}:${toString config.services.loki.configuration.server.http_listen_port}";
+      }
+    ];
+  };
+
+  services.nginx = {
+    virtualHosts = {
+      "_" = {
+        default = true;
+        locations."/" = {
+          proxyPass = "http://${config.services.grafana.settings.server.http_addr}:${toString config.services.grafana.settings.server.http_port}";
+          proxyWebsockets = true;
+        };
```

Comment:
This host block should be `"ghaflogs.vedenemo.dev"` instead of `"_"` for the domain name to work.

HTTPS should be forced:
```
enableACME = true;
forceSSL = true;
```

Acme needs to also be enabled with a separate configuration (to fetch https certificates):

```
security.acme = {
  acceptTerms = true;
  defaults.email = "trash@unikie.com";
};
```

Look at `hosts/himalia/configuration.nix` for example.

Reply:
Done.